spi_nand_flash/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+version: "0.0.3"
+description: Driver for accessing spi nand flash
+url: https://github.com/espressif/idf-extra-components/tree/master/spi_nand_flash
+dependencies:
+  idf: ">=5.2"
```

Comment:
Is there any reason to constrain this component to only IDF >=v5.2? From brief look, the APIs should also work with older IDF releases.